### DISCUSSION TITLE
[ Section 3 - 38 ] Duvida sobre API gateway

### DISCUSSION THREAD

**Discussion by chwiee** [chgasparoto/curso-aws-com-terraform#141] [Q&A]
Como podemos criar um endpoint composto? Por exemplo no curso ensinar a fazer o /todos eu pensei em fazer algo assim:

GET /todos 
POST /todos/new

no caso o /new ele é dentro do todos

**Comment by chgasparoto** [comment]
@chwiee tem um exemplo no youtube que eu gravei há um tempo atrás.

https://www.youtube.com/playlist?list=PLWQmZVQayUUK5581j2xYO6iTzooId50Lz
https://github.com/chgasparoto/youtube-cleber-gasparoto/blob/main/0005-terraform-serverless-pattern-rest-api/terraform/api.tf
